examples/sample-layers/package.json

```diff
@@ -4,10 +4,13 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.16.0",
-    "deck.gl": "3.1.0-beta.16",
+    "deck.gl": "^4.0.0-beta.2",
+    "extrude-polyline": "^1.0.6",
     "immutable": "^3.8.1",
-    "luma.gl": "3.0.0-alpha.4",
+    "lodash.flattendeep": "^4.4.0",
+    "luma.gl": "3.0.0-beta.9",
     "react": "^15.4.1",
+    "react-autobind": "^1.0.6",
```

Comment:
> BTW - this module is unmaintained and has some issues

Ah, good catch.

> I think most people are rolling w/ core-decorators autobind instead

For what it's worth, in v4 we've moved away from decorators and other advanced JS features in favor of a pure ES6 code base that can run without transpilation (modulo JSX) on the latest browsers. 

Ironically, we actually have our own small `autobind` implementation inside deck.gl (in order to minimize package dependencies) but we don't export it from the lib (in order to keep a small API surface).

It would of course be good if our examples follow best practices. Will have to investigate if core-decorators can be used without actual decorator syntax or if there is another suitable replacement. I guess we could just manually bind the methods in the constructor.